modules/windows-machineset-aws.adoc

```diff
@@ -81,6 +81,14 @@ $ oc get -o jsonpath='{.status.infrastructureName}{"\n"}' infrastructure cluster
 <3> Configure the compute machine set as a Windows machine.
 <4> Configure the Windows node as a compute machine.
 <5> Specify the AMI ID of a supported Windows image with a container runtime installed.
++
+[NOTE]
+====
+For disconnected clusters, the Windows AMI must have the EC2LaunchV2 agent version 2.0.1643 or later installed.
+
+For more information on the EC2LaunchV2 agent, see link:https://aws.amazon.com/blogs/modernizing-with-aws/upgrade-ec2launch-v2[Which AMIs have EC2Launch v2?] in the AWS documentation.
```

Comment:
I'd rather cite the official docs [1] than a blog post.


https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2launch-v2-install.html